### PR TITLE
Tutorial 1: split long exercise

### PR DESCRIPTION
**Pull request by christophlohrmann**
Description of changes:
- Make the really big last exercise into multiple small ones

### CODE REVIEW

**Comment by christophlohrmann** [comment]
@jngrad can you restart the failed test? It seems to be a random failure of the active matter tutorial

**Comment by christophlohrmann** [comment]
I don't understand the CI error

**Comment by KaiSzuttor** [comment]
```

686A call to mkdir was unable to create the desired directory:
687  Directory: /tmp/ompi.coyote.1000/pid.3549
688  Error:     No such file or directory
689Please check to ensure you have adequate permissions to perform
690the desired operation.
```

tmp might be full

File: doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb [hunk 1]
```diff
@@ -627,162 +606,297 @@
...
-    "\n",
-    "**Part II:**\n",
-    "* Extend the integration loop to include the calculation of the radial distribution function which describes how the density varies as a function of distance from a tagged particle. \n",
-    "  The radial distribution function (RDF) is averaged over several measurements to reduce noise.\n",
-    "  To this end add an observable <tt>rdf_obs</tt> for the radial distirbution function.\n",
-    "  Use min_r=R_MIN, max_r=R_MAX, n_r_bins=R_BINS and create a corresponding [<tt>espressomd.accumulators.MeanVarianceCalculator</tt>](http://espressomd.org/html/doc/espressomd.html#espressomd.accumulators.MeanVarianceCalculator)\n",
-    "  that gets updated every SAMPLING_INTERVAL timesteps.\n",
-    "  Plot the RDF using the mean of the accumulator.\n",
-    "  *Hint*: The radial coordinates can be obtained using the <tt>bin_centers()</tt> method of the <tt>espressomd.observables.RDF</tt> observable."
+    "* Integrate the system and measure the total and kinetic energy. Take ``N_SAMPLES`` measurements every ``STEPS_PER_SAMPLE`` integration steps.\n",
```
Comment (reinaual) [inline doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb:609]: Could be slightly more precise
```suggestion
    "* Integrate the system and measure the total and kinetic energy. Take a measurement every ``STEPS_PER_SAMPLE`` integration steps for a total of ``N_SAMPLES`` measurements.\n",
```

File: doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb [hunk 2]
```diff
@@ -627,162 +606,297 @@
...
+    "we may compute the actual temperature of the system via $k_B T= 2/(3N) \\langle E_\\text{kin}\\rangle$.\n",
+    "The temperature is fixed and does not fluctuate in the NVT ensemble! The instantaneous temperature is\n",
+    "calculated via $2/(3N) E_\\text{kin}$ (without ensemble averaging), but it is not the temperature of the system."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the first simulation run we picked ``STEPS_PER_SAMPLE`` arbitrary. To ensure proper statistics at short total run time, we will now calculate the ``steps_per_uncorrelated_sample`` which we will use for the rest of the tutorial."
```
Comment (reinaual) [inline doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb:667]:
```suggestion
    "In the first simulation run we picked ``STEPS_PER_SAMPLE`` arbitrary. To ensure uncorrelated measurements while keeping the total run time short, we will now calculate the ``steps_per_uncorrelated_sample`` which we will use for the rest of the tutorial."
```

File: doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb [hunk 3]
```diff
@@ -627,162 +606,297 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use only the data after the equilibration period in the beginning\n",
+    "warmup_time = 15\n",
+    "e_total = e_total[times > warmup_time]\n",
+    "e_kin = e_kin[times > warmup_time]\n",
+    "times = times[times > warmup_time]\n",
+    "times -= times[0]"
```
Comment (reinaual) [inline doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb:681]:
```suggestion
    "WARMUP_TIME = 15\n",
    "e_total = e_total[times > WARMUP_TIME]\n",
    "e_kin = e_kin[times > WARMUP_TIME]\n",
    "times = times[times > WARMUP_TIME]\n",
    "times -= times[0]"
```
this is a constant, right?

File: doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb [hunk 4]
```diff
@@ -627,162 +606,297 @@
...
+    "### Automated data collection\n",
+    "\n",
+    "As we have seen, it is easy to manually extract information from an **ESPResSo** simulation, but it can get quite tedious. Therefore, **ESPResSo** provides a number of data collection tools to make life easier (and less error-prone). We will now demonstrate those with the calculation of the radial distribution function."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Observables](http://espressomd.org/html/doc/analysis.html#using-observables) extract properties of the particles and calculate some quantity from it, e.g. the center of mass, the total energy or a histogram.\n",
```
Comment (reinaual) [inline doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb:871]:
```suggestion
    "[Observables](http://espressomd.org/html/doc/analysis.html#using-observables) extract properties from the particles and calculate some quantity with it, e.g. the center of mass, the total energy or a histogram.\n",
```

File: doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb [hunk 5]
```diff
@@ -627,162 +606,297 @@
...
+    "\n",
+    "[Accumulators](http://espressomd.org/html/doc/analysis.html#accumulators) allow calculation of observables while running the system and then doing further analysis. Examples are a simple time series or more advanced methods like correlators.\n",
```
Comment (reinaual) [inline doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb:873]:
```suggestion
    "[Accumulators](http://espressomd.org/html/doc/analysis.html#accumulators) allow the calculation of observables while running the system and then doing further analysis. Examples are a simple time series or more advanced methods like correlators.\n",
```

File: doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb [hunk 6]
```diff
@@ -627,162 +606,297 @@
...
-    "solution2": "hidden"
+    "solution2": "hidden",
+    "solution2_first": true
    },
    "source": [
-    "From the roughly exponential decay of the correlation function it dawns that only every second sample should be used to calculate statistically independent averages."
+    "**Exercise**\n",
+    "* Instantiate a [RDF](http://espressomd.org/html/doc/espressomd.html#espressomd.observables.RDF) observable\n",
+    "* Instantiate a [MeanVarianceCalculator](http://espressomd.org/html/doc/analysis.html#mean-variance-calculator) accumulator to track the RDF over time. Samples should be taken every ``steps_per_uncorrelated_sample`` steps.\n",
+    "* Add the accumulator to the [auto_update_accumulators](http://espressomd.org/html/doc/espressomd.html#espressomd.accumulators.AutoUpdateAccumulators) of the system for automatic update"
```
Comment (reinaual) [inline doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb:899]:
```suggestion
    "* Add the accumulator to the [auto_update_accumulators](http://espressomd.org/html/doc/espressomd.html#espressomd.accumulators.AutoUpdateAccumulators) of the system for automatic updates"
```

File: doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb [hunk 7]
```diff
@@ -792,99 +906,175 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we don't need an elaborate integration loop, instead the RDFs are calculated and accumulated automatically"
```
Comment (reinaual) [inline doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb:926]:
```suggestion
    "Now we don't need an elaborate integration loop anymore, instead the RDFs are calculated and accumulated automatically."
```

File: doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb [hunk 8]
```diff
@@ -627,162 +606,297 @@
...
-    "code_folding": [],
     "solution2": "hidden"
    },
    "source": [
     "```python\n",
-    "# Plot total energy and instantaneous temperature\n",
-    "import matplotlib.pyplot as plt\n",
+    "e_total_autocor = autocor(e_total)\n",
+    "corr_time = fit_correlation_time(e_total_autocor, times)\n",
+    "steps_per_uncorrelated_sample = int(np.ceil(3*corr_time/system.time_step))\n",
```
Comment (reinaual) [inline doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb:731]: do we want this `steps_per_uncorrelated_sample` to be an all capital const?

File: doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb [hunk 9]
```diff
@@ -792,99 +906,175 @@
...
-    "\n",
-    "# reduced units\n",
-    "T_star = TEMPERATURE / LJ_EPS\n",
-    "rho_star = DENSITY * LJ_SIG**3\n",
-    "\n",
-    "# expression of the factors Pi from Equations 2-8 with coefficients qi from Table 1\n",
-    "# expression for a,g\n",
-    "P = lambda q1, q2, q3, q4, q5, q6, q7, q8, q9: \\\n",
+    "rdf = rdf_acc.get_mean()\n",
+    "rs = rdf_obs.bin_centers()\n",
```
Comment (reinaual) [inline doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb:958]: isn't this name a bit short? Also i had to look for the plot label to understand the name. Personally i think `bin_centers` would be more expressive but the development from `r` to `rs` is already a step in the right direction.

**Comment by RudolfWeeber** [comment]
`radii` or `bin_centers`?

**Review by RudolfWeeber — APPROVED** [review]

**Pull request merged**